lib/solr/params.rb

```diff
@@ -19,6 +19,11 @@ def fq(field, value)
       self
     end
 
+    def facet_on(field)
+      @parameters[:"facet.field"] << field unless field.blank?
```

Comment:
```suggestion
      @parameters["facet"] = "on"
      @parameters[:"facet.field"] << field unless field.blank?
```
Does this simplify the `to_h` method? If not feel free to ignore.